src/components/sbb-navigation-section/sbb-navigation-section.tsx

```diff
@@ -181,13 +181,17 @@ export class SbbNavigationSection implements ComponentInterface {
         const firstFocusable = focusableElements[0] as HTMLElement;
         const lastFocusable = focusableElements[focusableElements.length - 1] as HTMLElement;
 
+        const actionToFocus = event.shiftKey
+          ? this._triggerElement
+          : (this._triggerElement.nextElementSibling as HTMLElement);
```

Comment:
I think it's a little bit risky to just us nextElementSibling as it can have some other elements, e.g. divider or whatever invisible span / element. What about searching for the next focusable element starting from trigger?